src_py/draw_py.py

```diff
@@ -434,6 +434,21 @@ def draw_aaline(surf, color, from_point, to_point, blend=True):
 def draw_line(surf, color, from_point, to_point, width=1):
     """draw anti-aliased line between two endpoints."""
     line = [from_point[0], from_point[1], to_point[0], to_point[1]]
+    if width!=1:
+        #trigonometry
+        #angle=math.atan2(*map(float.__sub__,destination,initial))%(math.pi/2)
+        #angle=min(angle,math.pi/2-angle)
+        #width=int(width/math.cos(angle))
+        #optimised by 1/cos(atan(x/y)=sqrt((x/y)**2+1)
+        #displacement=[(d-i)**2 for d,i in zip(to_point,from_point)]
+        #displacement.sort()
+        #if displacement[1]!=0.0:
+            #width=int(width*(displacement[0]/displacement[1]+1)**0.5)
+        #more optimised
+        (x,y)=[(d-i)**2 for d,i in zip(to_point,from_point)]
```

Comment:
`(x,y)=[(d-i)**2 for d,i in zip(to_point,from_point)]`
This took me a moment to understand. If I understand it correctly it is the same as 
```
x = (from_point[0] - to_point[0])**2  # order does not matter because of the square
y = (from_point[1] - to_point[1])**2 
```
This would be much more readable, not sure about performance.

I would suggest to rename the variables to `dx2` and `dy2` to indicate that it is a difference (like 'delta x') and the 2 for squared (or maybe even `dx_sqared` or `dx_sq` ).